### DISCUSSION TITLE
All the same ressources on the same graph

### DISCUSSION THREAD

**Discussion by dominiquearpin** [cdklabs/cdk-monitoring-constructs#437] [Q&A] [answered]
Hi team,

Presently, If I have 10 apigw, I got 10 graphs.  Its possible to have only 1 graph with all the 10 apigw if the have all the same threshould/alarms?

Regards,

**Comment by ghost** [comment, marked as answer]
Unfortunately, the current implementations are generally scoped to single resource instances. If you'd like something like that, you can implement [a custom segment](https://github.com/cdklabs/cdk-monitoring-constructs/blob/main/README.md#custom-monitoring-segments), although you can likely leverage the existing metric factory implementations.
